docs/imagemanifestv2.md

```diff
@@ -0,0 +1,113 @@
+# Image Manifest V2 
+
+This document outlines the format of of the V2 image manifest.  Image manifests describe the various constituents of a docker image.  Image manifests can be serialized to JSON format.
+
+
+## *Manifest* Field Descriptions
+
+Manifest provides the base accessible fields for working with V2 image format in the registry.
+
+- **`Name`** *string*
+
+	Name is the name of the image's repository
+
+- **`Tag`** *string*
+	
+	Tag is the tag of the image
+	
+- **`Architecture`** *string*
+
+   Architecture is the host architecture on which this image is intended to run
```

Comment:
Let's make it clear this is informative only right now, and not actively used by the engine.